diff.py

```diff
@@ -782,13 +786,23 @@ def do_diff(basedump, mydump):
                     branchless1 = out1
                     branchless2 = out2
                     out1, out2 = color_imms(out1, out2)
-                    branch1, branch2 = color_branch_imms(branch1, branch2)
+
+                    same_relative_target = False
+                    if branch_targets1[i1 + k] is not None and branch_targets2[j1 + k] is not None:
+                        relative_target1 = eval_line_num(branch_targets1[i1 + k]) - eval_line_num(line_num1)
+                        relative_target2 = eval_line_num(branch_targets2[j1 + k]) - eval_line_num(line_num2)
+                        same_relative_target = relative_target1 == relative_target2
+
+                    if not same_relative_target:
+                        branch1, branch2 = color_branch_imms(branch1, branch2)
```

Comment:
I feel like the check for branch1 == branch2 within color_branch_imms is a bit sketchy with this change... OTOH, it probably doesn't hurt in practice (might even help, in the non-`-e` case), and the `maybe_normalize_large_imms(original1) == maybe_normalize_large_imms(original2)` check is equally broken. So eh, let's go with this I guess.